scipy/stats/tests/test_distributions.py

```diff
@@ -7870,6 +7870,22 @@ def test_regression_tukey_lambda():
     assert_((p[2] == 0.0).any())
 
 
+def test_tukeylambda_support():
```

Comment:
I'd be willing to review a commit that brought all the scattered `tukeylambda` tests into one class.

Reply:
I almost did that originally, so I'll do that now.  I'm seeing some quirks (possible bugs) in `test_regression_tukey_lambda` that I'll clean up while I'm at it.

Reply:
The bug is the test `assert_(~np.isnan(p).all())`.  The intent was almost certainly to check that there are no nans in `p`, but all it takes for that test to succeed is at least one non-nan value in `p`.  I'll change it to `assert np.isfinite(p).all()`.